app/assets/stylesheets/flights.scss

```diff
@@ -1,3 +1,161 @@
-// Place all the styles related to the flights controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
+// Style for will pagination
+.digg-pagination {
+  background: white;
+  cursor: default;
+
+  a, span, em {
+    padding: 0.2em 0.5em;
+    display: block;
+    float: left;
+    margin-right: 1px;
+  }
+
+  .disabled {
+    color: #999999;
+    border: 1px solid #dddddd;
+  }
+
+  .current {
+    font-style: normal;
+    font-weight: bold;
+    background: #2e6ab1;
+    color: white;
+    border: 1px solid #2e6ab1;
+  }
+
+  a {
+    text-decoration: none;
+    color: #105cb6;
+    border: 1px solid #9aafe5;
+
+    &:hover, &:focus {
+      color: #000033;
+      border-color: #000033;
+    }
+  }
+
+  .page_info {
+    background: #2e6ab1;
+    color: white;
+    padding: 0.4em 0.6em;
```

Comment:
Properties should be ordered background, color, margin-bottom, padding, text-align, width